Basic part2

komarikl opened the pull request.

DaniilPonomarev579 committed:
fix html and css tasks

kanstudio committed:
Update README.md

Some grammatical corrections.

ifedyukin committed:
Merge pull request #16 from WebPurple/fix/html&css-tasks

fix html and css tasks

ifedyukin committed:
Merge pull request #17 from kanstudio/master

Update README.md

kutur15 committed:
draft for the second HW

Seagull54 committed:
changed ex3 task 5 test to correctly represent test.

kutur15 committed:
task02 part01 attempt02

kutur15 committed:
task02 part01 attempt03

kutur15 committed:
task02 part02 attempt1

kanstudio committed:
Update ex3_js-objects-part1.test.js

В строке:
expect(objCopy).toEqual(obj);
аргумент функции expect - ожидаемый результат, а аргумент метода toEqual - исходный тестируемый.

В случае, если решение задачи верное, проверка пройдёт корректно в любом случае, т.к. при проверке равенства без разницы - в каком порядке идут сравниваемые объекты.

Но если задача решена неверно, то в логах Travis в описании ошибки будет указано, что получен (received) верный результат, а ожидается (expected) неверный. Это может ввести в заблуждение при отладке неработающего кода.

ifedyukin committed:
Merge pull request #35 from kanstudio/master

Update ex3_js-objects-part1.test.js

ifedyukin committed:
Merge pull request #32 from LordDmitry/ex3task5-fix

Ex3 test fix

kutur15 committed:
task02 part02

eugrdn committed:
refactor(exercises): updated names, added oop exercise

kutur15 committed:
task02 part02

kutur15 committed:
task02 part02 attempt04

kutur15 committed:
task02 part02 attempt05

kutur15 committed:
task02 part02-03 attempt01

kutur15 commented:
1. выводит в консоль каждый элемент массива и общее число элементов
2. выборка из массива четных нечетных и нулей

kutur15 committed:
task02 part03 attempt6

kutur15 committed:
task02 part03 attempt7

ifedyukin committed:
Merge pull request #36 from WebPurple/refactor/exercises

refactor(exercises): updated names, added oop exercise

kutur15 committed:
task02 part03 attempt7

kutur15 committed:
task02 part03

Commit:
add task 2 for js-context

Commit:
remive empty lines

kutur15 committed:
task02 part3-4

kutur15 commented:
выведет в консоль количество четных и нечетных элементов
поиск в массиве одинаковых элементов,

ifedyukin committed:
Merge pull request #39 from WebPurple/js_context_add_task_2

add task 2 for js-context

kutur15 committed:
task02 part5

kutur15 commented:
поиск максимального в массиве

kutur15 committed:
task02

kutur15 committed:
task02

kutur15 committed:
task02

kutur15 committed:
task02

kutur15 committed:
task02

kutur15 committed:
task01-02

kutur15 commented:
задача 02 часть1-5

kutur15 committed:
task02 part06

kutur15 committed:
task02-part06

kutur15 committed:
task02-part06

kutur15 commented:
вывод простого/составного числа

kanstudio committed:
Update .eslintrc

kanstudio committed:
Update ex4_js-objects-part2.test.js

ellipsis - многоточие, это 1 символ, не равен трём точкам

kanstudio committed:
Update README.md

Изображённый в описании символ «…» - многоточие, это 1 символ, не то же самое, что 3 точки (3 символа)

ifedyukin committed:
Merge pull request #41 from kanstudio/master-clear

Update .eslintrc

ifedyukin committed:
Merge pull request #42 from kanstudio/patch-1

Update ex4_js-objects-part2, task-07

kanstudio committed:
Update ex4_js-objects-part2.test.js

Вызов функции code('abbac'); должен быть внутри теста it();
Иначе код не запускается и количество запусков console.log не считается.

eugrdn committed:
feat(w3c-validators): added w3c validation for html/css through the tests

eugrdn committed:
feat(w3c-validators): README updated

eugrdn committed:
Merge pull request #43 from kanstudio/patch-2

Update ex4_js-objects-part2.test.js

ifedyukin committed:
Merge pull request #44 from WebPurple/feat/w3c-validators

feat/w3c-validators

kutur15 committed:
task3 part1

kutur15 commented:
ввод данных в объект

kutur15 committed:
task3 part2

kutur15 committed:
task3 part2

kutur15 committed:
task3 part2

kutur15 commented:
функцию, принимает в качестве аргумента объект и выводит в консоль все ключи

kutur15 committed:
task3 part3

kutur15 commented:
функция, строка, объект, возвратить значение

kutur15 committed:
task03 part04

kutur15 committed:
task03-4

kutur15 committed:
task03

kutur15 committed:
task03 part4

kutur15 committed:
task03

kutur15 committed:
task03

kutur15 commented:
функция, строка, объект, возвратить значение

kutur15 committed:
task03

kutur15 committed:
task3

kutur15 committed:
task3

kutur15 commented:
функцию, которая принимает объект и создает его копию

kutur15 committed:
task03

kutur15 commented:
копирование объекта с рекурсией

kutur15 committed:
Merge branch 'master' of https://github.com/WebPurple/external-courses into basic_part2

kutur15 committed:
remove 3 HW tasks

kutur15 committed:
test

kutur15 committed:
lock

kutur15 committed:
js

kutur15 committed:
test

kutur15 committed:
Merge branch 'master' into basic_part2
